.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
make sure to avoid commits of `.vscode` and `.DS_Store` files.
please remove them and also add to gitignore.